modules/shortcodes/vr.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+function jetpack_vr_viewer_get_viewer_url_params( $params ) {
+	$url_params = array();
+
+	if ( isset( $params['rotation'] ) ) {
+		$url_params['rotation'] = intval( $params['rotation'], 10 );
+	}
+
+	if ( isset( $params['view'] ) && in_array( $params['view'], array( 'cinema', '360' ), true ) ) {
+		$url_params['view'] = $params['view'];
+	}
+
+	if ( $params['preview'] ) {
+		$url_params['preview'] = 1;
+	}
+
+	if ( isset( $params['url'] ) ) {
+		return array_merge( $url_params, array( 'url' => $params['url'] ) );
+	} else if ( isset( $params['guid'] ) ) {
+		return array_merge( $url_params, array( 'guid' => $params['guid'] ) );
+	}
+
+	return false;
+}
+
+function jetpack_vr_viewer_iframe_padding( $view ) {
+	if ( $view === '360' ) {
+		return '100%'; // 1:1 square aspect for 360
+	}
+
+	return '50%'; // 2:1 panorama aspect
+}
+
+function jetpack_vr_viewer_get_html( $url_params ) {
+	$iframe = add_query_arg( $url_params, 'https://vr.me.sh/view/' );
+
+	$rtn  = '<div style="position: relative; max-width: 720px; margin-left: auto; margin-right: auto; overflow: hidden;">';
```

Comment:
Instead of forcing the width here, what would you say about using `$content_width` to define the maximum width of the embeds?